packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -400,12 +402,25 @@ class _AuthenticatorState extends State<Authenticator> {
     });
   }
 
+  Future<void> _setUpHubSubscription() async {
+    // the stream does not exist until configuration is complete
+    await Amplify.asyncConfig;
+    _hubSubscription = Amplify.Hub.listen([HubChannel.Auth], (event) {
+      switch (event.eventName) {
+        case 'SIGNED_OUT':
+          _stateMachineBloc.add(const AuthNavigateToSignIn());
```

Comment:
Can we use `AuthChangeScreen`?

Reply:
I did not realize that existed. Updated.